### PR TITLE
Fix regression: Go to Definition stop working for variables inside command

### PR DESCRIPTION
**Pull request by kyklish**
Regression: all `OutputVar...` stop detecting like variables (see example string below):

`MouseGetPos, OutputVarX, OutputVarY, OutputVarWin, OutputVarControl`

Changes proposed in this pull request:

- restore original `CodeUtil.purify()` function (used not only in `FormatProvider` but for parsing whole script to detect symbols, my bad :(, sorry Mark)
- create separate `purify()` function for `FormatProvider`

Notifying @mark-wiemer

### CODE REVIEW

File: src/common/codeUtil.ts [hunk 1]
```diff
@@ -7,227 +7,13 @@ export class CodeUtil {
         if (!original) {
```
Comment (mark-wiemer) [inline src/common/codeUtil.ts:7]: Let's keep the `test/suite/common/codeUtil.test.ts` and have it test the behavior of `CodeUtil.purify`. This function should have some clear tests as it's pretty core to the functionality of our extension :)

File: src/providers/formattingProvider.utils.ts [hunk 2]
```diff
@@ -83,6 +83,237 @@ export function hasMoreOpenParens(line: string): boolean {
...
+        if (original.search(regExp) !== -1) {
+            cmdTrim = original.replace(regExp, '$1');
+            break;
+        }
+    }
+    //
+    let pure = cmdTrim
+        .replace(/".*?"/g, '""') // replace string literals with empty string literal
+        .replace(/{.*}/g, '') // remove matching braces
+        .replace(/\s+/g, ' ') // collaps all spaces and tabs to single space
```
Comment (mark-wiemer) [inline src/providers/formattingProvider.utils.ts:311]:
```suggestion
        .replace(/\s+/g, ' ') // collapse all spaces and tabs to single space
```

**Review by mark-wiemer — CHANGES_REQUESTED** [review]
Coolio, just a few more tests and this looks good :)

File: src/common/codeUtil.ts (hunk 1 above)
Comment (kyklish) [reply src/common/codeUtil.ts:7]: Done.

**Review by mark-wiemer — APPROVED** [review]

**Pull request merged**